### PR TITLE
Adds a new wall cabinet for medkits and makes empty basic medkits printable at the medical protolathe.

### PR DESCRIPTION
**Pull request by shellspeed1**
## About The Pull Request

This pr adds a new medkit wall cabinet to the game that works the exact same way as a fire extinguisher cabinets (I mean it's the code for those repurposed). It also enables medbay to print empty medkits to put in them as well as to hold supplies for doctors. Both are unlocked with biological technology researched and are printable at the medical protolathe. If placed by mappers it will contain a basic medkit. Also thank you to BrokenOculus for making sprites.
## Why It's Good For The Game
This PR is meant to encourage medbay to distribute supplies around the station to make it easier to handle emergencies and give them something to do to help make medbay a little less reactive and a bit more proactive. Also helps to make supplies a bit more visible since people have a tendency to fail to look inside the chem fridge for medication the chemist has made.

## Changelog
:cl:
add: Adds  medkit wall cabinets and empty medkits to the medical protolathe.
add: Medkit wall cabinents and empty medkits are now part of biological technology.

/:cl:

### CODE REVIEW

**Comment by shellspeed1** [comment]
AAAAAAAAAA merge conflicts, what do?

**Comment by ArcaneMusic** [comment]
I think I might know why, as in my Medical Kiosk PR, I added to Biotech as well. Try adding that in to all_nodes.dm . It's probably being difficult because of that.

**Comment by shellspeed1** [comment]
Yeah figured it out, working on it.

**Comment by shellspeed1** [comment]
fixed

**Comment by Arturlang** [comment]
Perhaps make it a subtype of the fire extinguisher locker so you dont have to copy code as much.

**Comment by Tlaltecuhtli** [comment]
infinite medibots? heh

**Comment by shellspeed1** [comment]
> 
> 
> infinite medibots? heh

Assuming the chemists can be bothered to make plastic for once, I guess. Would only be basic medkits though so eh.

**Comment by shellspeed1** [comment]
> 
> 
> Perhaps make it a subtype of the fire extinguisher locker so you dont have to copy code as much.

I sadly lack the knowledge of how to do that.

**Comment by shellspeed1** [comment]
whacked wrong button.

File: code/game/objects/structures/medkit.dm [hunk 1]
```diff
@@ -0,0 +1,164 @@
+///Reuses fire extinguisher cabinet wall mount code to create an equivalent for medkits.
```
Comment (nemvar) [inline code/game/objects/structures/medkit.dm]: BAD. I haven't double checked but if this is literally just duplicated over from extinguishers you better fix this.

**Comment by nemvar** [comment]
The concept for making this a subtype is basically the same as copy pasting, only it results in less lines.
Basically you just replace any instance of fire extinguishers and/or the icon being hardcoded with a local variable and change said variable in your subtype.

**Comment by shellspeed1** [comment]
Alright, I'll see if I can redo it as a subtype than.

**Pull request closed without merging**